RockLib.Messaging/ReceiverExtensions.cs

```diff
@@ -89,7 +105,9 @@ public Task OnMessageReceivedAsync(IReceiver receiver, IReceiverMessage message)
                     _onMessageReceived(receiver, message);
                     source.SetResult(0);
                 }
+#pragma warning disable CA1031 // Do not catch general exception types
                 catch (Exception ex)
```

Comment:
Why do you not want to catch a specific exception here?  Or are you not updating it because it's marked obsolete?

Reply:
The latter. The whole sync approach should go away.